### DISCUSSION TITLE
Issues after realise of 4.9.2

### DISCUSSION THREAD

**Discussion by pwbourne** [microsoft-search/pnp-modern-search#3189] [Q&A]
I am getting an error 
“[HTTP]:400 - Bad Request [CorrelationId]:5360d0a0-7038-7000-02de-f4b29914a5e1 [Version]:16.0.0.23926”

I was off for 1 week so all was working fine 1 week ago, I return today and get the above error. I noticed the version was updated so uploaded the version to Sharepoint but still the same issue.

Tried using the SharePoint search query tool but still get a 400 error when trying to search the site. I am not sure what steps to try next

**Comment by wobba** [comment]
API issues should be reported via Microsoft support. Try to repro using classic search experience first.

**Reply by pwbourne** [reply]
Thank you for your quick reply, Some more information some parts of the web part are no longer responding as expected for example the selected properties no loungers shows a drop down list just a buffering circle, also when using a detailed list under manage columns the column value drop down no longer works.

I get this even if I add a new web part. I can only seem to get a people search to work with out a 400 error.

**Reply by pwbourne** [reply]
Some more information:
I have removed Pnp search and API permissions granted and re-added. So up up a new site
Added a Search results web part and connected to a search box on the same page.
Made no other updates to the configuration.
What ever  I type in the search box results in a 400 error

**Reply by wobba** [reply]
So you are using the Graph provider for the web part, not SharePoint Search API? If things worked, and now they failed for the same setup, then it's likely not a web part issue. Thus a support issue with Microsoft on API's, not the web parts.
